### PR TITLE
DRYD-1035: Add checksum fields to media handling and collectionobject

### PR DESCRIPTION
**Pull request by mikejritter**
Jira Ticket: https://collectionspace.atlassian.net/browse/DRYD-1035

Also added to collectionobject per Jira Ticket: https://collectionspace.atlassian.net/browse/DRYD-1061

### CODE REVIEW

**Review by ray-lee — APPROVED** [review]
Looks good!

**Comment by ray-lee** [comment]
@mikejritter This just needs conflicts resolved, then I can merge it.

**Comment by mikejritter** [comment]
@ray-lee I just pushed changes for this and the application layer to resolve the conflicts. Did a quick check to make sure everything is still saving, though I just realized I didn't add any tests for the ui layer. Should be good to go, lmk if you want me to add tests before merging.

**Pull request merged**
